shared/reducers/unlock-folders.js

```diff
@@ -20,7 +27,8 @@ const initialState: State = {
   closed: true,
   phase: 'dead',
   waiting: false,
-  devices: null,
+  devices: [],
```

Comment:
why? I think it's useful to know when you don't have something yet vs an empty set.

If you can handle an empty set and not having something the same way I agree it's preferred, but I think it's better to be explicit when we won't have something.